cli/js/io.ts

```diff
@@ -70,9 +70,13 @@ export interface ReadWriteCloser extends Reader, Writer, Closer {}
 // https://golang.org/pkg/io/#ReadWriteSeeker
 export interface ReadWriteSeeker extends Reader, Writer, Seeker {}
 
-export async function copy(src: Reader, dst: Writer): Promise<number> {
+export async function copy(
+  src: Reader,
+  dst: Writer,
+  bufSize?: number
```

Comment:
Wrap it in `options` object just like `iter` and `iterSync` - then we might extend the API to take `buffer` instead of `bufSize` if user prefers to pass owned buffer

Reply:
Done :). I like it better with an object.